nautobot/core/tests/test_filters.py

```diff
@@ -851,7 +851,7 @@ def setUpTestData(cls):
                 rack=racks[0],
                 position=1,
                 face=dcim_choices.DeviceFaceChoices.FACE_FRONT,
-                status=device_status_map["active"],
+                status=device_status_map["Active"],
```

Comment:
let's use an arbitrary item from `device_statuses` instead of hard-coded status names here and below